Packs/PaloAltoNetworks_IoT3rdParty/Integrations/PaloAltoNetworksIoT3rdParty/README.md

```diff
@@ -0,0 +1,163 @@
+Use Palo Alto Networks IoT 3rd Party to integrate Palo Alto Networks IoT solution with 3rd party products such as SIEM, Cisco ISE, Service Now etc.
```

Comment:
Use the Palo Alto Networks IoT 3rd Party integration to get devices, alerts, and vulnerabilities from PANW IoT Cloud.